### ISSUE TITLE
licensing is unclear

### ISSUE THREAD

**Issue by jonassmedegaard** [minorninth/libresample#6]
This project is dual-licensed, which can mean either of two things:
 a) both licenses are used at once - user must comply with **both** sets of requirements
 b) either licenses are used at a time - user can choose which **one** set of requirements to comply with

Which is it here?  A strict reading of the text seems to indicate a) but I suspect that the intent is b).

**Comment by emilazy** [comment]
I’m also curious if the BSD relicensing applies to the original code by Julius Smith or just the further changes.

**Comment by minorninth** [comment]
Sorry I missed this comment earlier. The answer is definitely (b) - you have the right to use the software under the terms of either license, your choice.

**Comment by minorninth** [comment]
I hope this makes it more clear.

https://github.com/minorninth/libresample/commit/cb18cc0f2fd106e48ba90f2fa705cf788f1f4777

**Comment by minorninth** [comment]
> I’m also curious if the BSD relicensing applies to the original code by Julius Smith or just the further changes.

Yes, Julius Smith explicitly consented to the dual-licensing. All of the code here is covered.
